ops/docker/hardhat/package.json

```diff
@@ -6,7 +6,7 @@
   },
   "license": "MIT",
   "dependencies": {
-    "dotenv": "^10.0.0",
-    "hardhat": "^2.9.6"
+    "dotenv": "^16.0.0",
+    "hardhat": "^2.12.0"
```

Comment:
CI issues probably caused by bumping hardhat. Need to make sure we use the same version of hardhat everywhere in the monorepo.

Reply:
Roger, will revert.